forms.rst

```diff
@@ -11,7 +11,7 @@ Forms
 
 Dealing with HTML forms is one of the most common - and challenging - tasks for
 a web developer. Symfony integrates a Form component that makes dealing with
-forms easy. In this article, you'll build a complex form from the ground up,
+forms straightforward. In this article, you'll build a complex form from the ground up,
```

Comment:
Straightforward falls in the same category as easy/simply etc, so I don't think this is an improvement

Reply:
" Symfony integrates a Form component that helps you dealing with forms" ?